targets/PythonSdk/source/source/PlayFabManualTest.py

```diff
@@ -0,0 +1,79 @@
+# idea here is to include PlayFabSettings, set them,
```

Comment:
Let's change this comment to a docstring.
A multiline docstring is defined like this:
"""
this is a
multiline
docstring
"""
Python interprets that a certain way, and applies it to the definition of the module, obtainable by reflection.

Let's also add the basic philosophy of this set of tests, and describe that the current iteration of the SDK is synchronous calls only